notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>Discussion regarding authorship info for enterprise hub recipes (internal): <a href="https://huggingface.slack.com/archives/C06AMDKKDNF/p1719999721983139" target="_blank">https://huggingface.slack.com/archives/C06AMDKKDNF/p1719999721983139</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>